components/dsp_processor/dsp_processor.c

```diff
@@ -57,42 +58,34 @@ void setup_dsp_i2s(uint32_t sample_rate, bool slave_i2s)
     .tx_desc_auto_clear = true                                              // Auto clear tx descriptor on underflow
   };
 
-  i2s_pin_config_t pin_config0 = {
-    .bck_io_num   = CONFIG_MASTER_I2S_BCK_PIN,
-    .ws_io_num    = CONFIG_MASTER_I2S_LRCK_PIN,
-    .data_out_num = CONFIG_MASTER_I2S_DATAOUT_PIN,
-    .data_in_num  = -1                                                       //Not used
-  };
+  i2s_pin_config_t pin_config0;
+  get_i2s_pins(I2S_NUM_0, &pin_config0);
 
   i2s_driver_install(0, &i2s_config0, 7, &i2s_queue);
   i2s_zero_dma_buffer(0);
   i2s_set_pin(0, &pin_config0);
-  gpio_set_drive_capability(CONFIG_MASTER_I2S_BCK_PIN,0);
-  gpio_set_drive_capability(CONFIG_MASTER_I2S_LRCK_PIN,0);
-  gpio_set_drive_capability(CONFIG_MASTER_I2S_DATAOUT_PIN,0);
-
-  i2s_config_t i2s_config1 = {
-    .mode = I2S_MODE_SLAVE | I2S_MODE_TX,                                   // Only TX - Slave channel
-    .sample_rate = sample_rate,
-    .bits_per_sample = bits_per_sample,
-    .channel_format = I2S_CHANNEL_FMT_RIGHT_LEFT,                           // 2-channels
-    .communication_format = I2S_COMM_FORMAT_I2S,
-    .dma_buf_count = 8,
-    .dma_buf_len = 480,
-    .use_apll = true,
-    .fixed_mclk = 0,
-    .tx_desc_auto_clear = true                                              // Auto clear tx descriptor on underflow
-  };
-
-  i2s_pin_config_t pin_config1 = {
-    .bck_io_num   =  CONFIG_SLAVE_I2S_BCK_PIN,
-    .ws_io_num    =  CONFIG_SLAVE_I2S_LRCK_PIN,
-    .data_out_num =  CONFIG_SLAVE_I2S_DATAOUT_PIN,
-    .data_in_num = -1                                                       //Not used
-  };
+  /* is this needed?
```

Comment:
I've commented this section to prevent using directly CONFIG_MASTER_I2S_XXX variables here, not sure if these statements are still needed in this version of the code.

Reply:
Still needed in an non ADF environment - Think i need to do a menuconfig to enable ADF or IDF compatibility